torch/nn/parallel/distributed.py

```diff
@@ -967,9 +969,9 @@ def _run_ddp_forward(self, *inputs, **kwargs):
                 self.device_ids[0],
                 self.use_side_stream_for_tensor_copies
             )
-            return module_to_run(*inputs[0], **kwargs[0])
+            return self._module_to_run_forward(*inputs[0], **kwargs[0])
```

Comment:
Discussed this offline, but the main issue here is that we  would not like to switch the call from `module(inputs)` to `module.forward(inputs)`. The main reason is because the former calls the the `__call__` of the nn.Module while the latter calls `forward`, and there is not necessarily equivalence between the two.